zebra-chain/src/orchard/keys/tests.rs

```diff
@@ -0,0 +1,70 @@
+#![allow(clippy::module_inception)]
+use super::*;
+
+#[cfg(test)]
+use proptest::prelude::*;
+
+#[cfg(test)]
+impl Arbitrary for TransmissionKey {
```

Comment:
We'll want Arbitrary for TransmissionKey in the arbitrary module, so it's available for proptest-impl.

This can happen as a follow-up.